tests/unit/hmsl/test_hmsl_utils.py

```diff
@@ -0,0 +1,60 @@
+from unittest.mock import MagicMock
+
+import pytest
+from pygitguardian.client import GGClient
+from pygitguardian.models import JWTResponse
+
+from ggshield.core.config.config import Config
+from ggshield.hmsl.utils import get_token, load_token_from_disk
+
+
+@pytest.fixture
+def hmsl_no_env_vars(monkeypatch):
+    monkeypatch.delenv("GITGUARDIAN_SAAS_URL", raising=False)
+    monkeypatch.delenv("GITGUARDIAN_SAAS_API_KEY", raising=False)
+    monkeypatch.delenv("GITGUARDIAN_HMSL_URL", raising=False)
+    monkeypatch.delenv("GITGUARDIAN_HMSL_AUDIENCE", raising=False)
+
+
+@pytest.fixture
+def gim(monkeypatch):
```

Comment:
Took me a while to understand what this fixture does. Can you rename it to `create_jwt_mock`?